assets/application.js

```diff
@@ -18,7 +18,14 @@ require('govuk-template')
 require('javascripts')
 require('./components')
 require('./components/show-hide-content/show-hide-content-init')
-var GOVUK = require('stageprompt')
 
-// initialise stageprompt for Analytics
-GOVUK.performance.stageprompt.setupForGoogleAnalytics()
+var stub = function() {console.log('This feature has been removed, please use Tracking Consent instead.')};
+window.GOVUK.Analytics = stub
```

Comment:
Where is window.GOVUK defined?

Reply:
It's defined in a number of different things that are borught in in the imports but I'm now being explicit about invoking it.